apps/dex/data.yaml

```diff
@@ -37,14 +37,13 @@ deploy_code: |
             namespace: dex
             values: |
               dex:
-                dex:
```

Comment:
Ok, thank you, we will merge this after revert of the example change.

Reply:
All set.